packages/core/src/components/menu/menuItem.tsx

```diff
@@ -76,8 +85,9 @@ export class MenuItem extends AbstractPureComponent<IMenuItemProps> {
                 target={this.props.target}
             >
                 <Icon iconName={this.props.iconName} />
+                <span className={textClasses}>{this.props.text}</span>
```

Comment:
back to the topic of our public CSS API! this is very similar to what I had a few months ago--it was considered breaking then, so it's technically breaking now since you're adding a span. if you remove the span from this current PR, this is what it looks like:
![image](https://user-images.githubusercontent.com/199754/35761178-a0a6feba-083a-11e8-85c6-ff2f3d016fe9.png)
![image](https://user-images.githubusercontent.com/199754/35761172-8e46ca02-083a-11e8-836c-da17d8e09fce.png)

Reply:
this implementation is not an API break, though! i added an element but only inside the React component. you can of course easily break the appearance by mucking in the console--that doesn't mean it's an API break.

you'll notice, most importantly, that all the CSS markup examples still look perfect, like before. **I made no external changes to the CSS or React APIs.**

folks who use only `MenuItem` and not `.pt-menu-item` will have a seamless upgrade experience (unless they have custom styles, of course).